Lab2.ipynb

```diff
@@ -0,0 +1,739 @@
ivJ518gt38/AEZHO9F33Ezk1luJ3PyOWfN821KALIXH0jKcGy73qQ6QS5uXukuTBMhLqjTltTo8jp1z14nzlXBZNbXYpdy7utatTFbtCzbLDXkEuaJdvu6yHDDTbsgczRQYLYXM8nYlhNrOxXOC3/C5obIUIMtTZCvbLRE/zWG/2w77aY5YxAKm/DdkBpGgWMdMDIov/vQxWj4kQVEIIRqtcOECqRdeJLV1K6kXXsAeHQWlCK5fT/SWm4ncciuhjRtQxuy6LmgwM8jR0aPlAFlaj+RGyn1iVqwcHJe3LGd583KWtyynJTg7QvSM4ziQHqyqTJ6tX6Wsd/2kFXFDY9M8iJeWbmia767j8yDYJGFS1KW1JpW3y9XK6gBZW70cV81MF0jkihc9r+lT5XDZEnaDZEvYDZxu291uClte261mBq3Z9d/bmUKCYh0zOig+sIpH/1qCohBCzATatsnu3Uty61ZSW58ns3s3OA5Gc7M7RfX224neegtG0+y826jWemIFctRtV18D2RHqKAfHFa0rWN68nCXNSwgY8sinK6KQda+bLFUkx7wAOXbGW7yAOf7aSSviBcluiM+vHywlTIppKtoOo5kCw2k3SA6nCgyn84ykvbU3Xba03+2bJ1u4+A1/gpavJli2hP1emKwETreKWQmcTSG5BvONzLrnKCql3g08BHQCn9VaP62UWgJ8HGjSWr+noQMUQggxayjDILRhA6ENG+j4zd/EHh0l9eKLJH/8HMmf/MR9dqNhENq8iejttxO74w78y5bNmqlVSinaQm20hdq4Ye4N5f2lR3n0DvdyaPgQh0YOcWj4EF8/8HXyTh5wH+PRE+9hRcuKcuVxecty5kXn4VPyKIBLygpCyyJ3uRi76IXJszB6ujZIjp6BIz9yj48Pk/5opQIZn1c/WAbiEiZFmWn4aIsGaItO7w9F2YJdGyir1iPpPMNV6wPnx8pTaCebIhsPmuUpsC3e3WLLlctIpbJZqly2RPxE/HL9JTSgoqiU+jLwMHBBa72uav/9wP8ADOCLWutPT+FcLcBfaK0/UrXvm1MJilJRFEII8VZp2yb7+usknnuO5I+fK1/baHV3E73jDqJ33E74xhvxBa6dqlrRKXJy7CS9I16AHD5E73AvZ5Jnyn3CZphlLcvK4XFFywpWtKyQZ0DOBHbBvXPr6BkY88Lk6JlxlcnzwLjfH/3RqqmtpUA531sWuGsr1JCPJGY3x9EkckVGvVB5sWBZXbkcSU0+PdYyFC1hP62RSqAsbTeH/bRGKuGyNeK/qsPljJp6qpS6DUgC/1AKikopA+gF7gVOA68A78cNjZ8ad4oPa60veK/7S+CftNavVZ1fgqIQQoiGKJw/T/K5n5B87jlS27ahMxlUMEjkppvcKap33I41Z06jh9kQqUKKwyOHy8GxVIWsnr7aGeqsBMfWFaxqWcWipkWYvqtyAtTsZRe8m+2cqV+ZHDvrhs3xYTLc7gbG5gWV8FgOkgsg0i5VSXHFFLzpseUgmXKnxA6nKgFzKJX3wmdpf56LFS/9ho/msHXRcHn9whY2Lmi+sh9yCmbU1FOt9U+UUovG7b4BOKy1PgqglPoG8KjW+lO41ccayo3rnwa+Vx0ShRBCiEay5syh5X0/R8v7fg4nlyP98svuFNXnniP57LMABFatcqeo3nkHwQ0bUL5rYwpmxIqwsWMjGzs2lvdprenP9FeCoxcev7b/axS8O336fX6WtSxjVesqVrasZGXrSla2rCTqv7qedzmrGJYb9poXXLxPMe9eEzl6GkZPectpGDkF/b1w+BkopGtfYwarwuN8aOoZtz0fzGunOi8uL8vw0R4N0D6N6bGlx5QMeSGyXph09xfo7UvWhMvfuWfFjAyKk5kpf6KbB5yq2j4N3DhJ/98C7gGalFLLtNafV0q1AX8KbFZK/YEXMmsopT4KfBSgp6fnkg3+krk27yskhBCzki8QIHrrrURvvRX9iY+TP3LEDYw/fo7BL36RwS98AaOjndgddxK9+y4iN910TU1RBff6x85wJ53hTm6Zd0t5f8EpcHz0OAeGDtA73MuBoQM8e/JZvnXoW+U+86Pz3fDoBcdVrauYE5lzVU79mpVMP7QsdJd6tIbMcCVAjp6GkZOV9qEfutdLjhftqp3O2uQF1lI71CJVSXHZ+HzKffxH2GJxe2RKrymFy6vxsuyZEhTr/Rt90diktf5b4G/H7RsEfmOyN9Fa/z3w9+BOPZ3+MIUQQojpU0oRWLaMwLJltH3kI9ijoyR/spXEj55h7IknGPm3f0OFw0Rvvpno3XcRvf32WfPMxjfD8lnlG9+UaK25kL7AweGDHBw6yIGhAxwcPsgzJ59Be78yxP1xVrWuYkXLCla1rmJV6yqWNC3BMqxGfRRxMUpBuNVd5m6s36eYq0xvLVUjS8Gybw/0PgXFbO1rrEil+ti8AJp7oHmht+6BSCdcI1V8MTOUwuXVaKYExdNA9fyF+cDZBo1FCCGEuKyMpiaa3vUwTe96GCefJ/3SSySeeYbkj54l8YMfgGEQvu46YvfcTfTuu/HPn9/oITecUoquSBddkS5um39beX+6kKZ3uNcNj8MH6B3q5Zu93yRruwHC9JksbVrKytaV5fC4snUlcX+8UR9FTJUZgNYl7lKP1u4zJkdPeSHydO1U13O7ID1Q+xojUBUgeyRICjGJhjxH0btG8fGqm9mYuDezuRs4g3szm1/QWu+9XGOYkTez2fYYLb8iN7MRQohrlXYcsnv3uqHxmR+RO3QIgMCKFUTvvovYXXcTXLdWple+AduxOZE4wcGhg+UAeXDoIAOZSmiYH53PmrY1rG5bzZpWd90SvHaruLNWPuWGyJGTMHLCW1ctFw2SC+uHyWinTG0Vs8pMu+vp14E7gHagD/ik1vpLSqkHgb/BvdPpl7XWf3o5xzEjg+KL36blw38gQVEIIQQA+VOnyqExvX07OA5mVxfRu+4kdvc9RG54G8rvb/QwrxoDm
```

Comment:
Problem 4D:
- typo: realtionship --> relationship
- by log relationship in your explanation did you mean linear?
- this last part is confusing me a little bit; maybe explain in further detail how this impacts the sigma values. I think what your plot is saying is as we increase n (amount of pixels) the threshold for 5-sigma detection linearly increases, so for a huge amount of pixels, we need a higher threshold? maybe include a deeper discussion to confirm